ui/mods/msu/css/popup.css

```diff
@@ -1,4 +1,5 @@
 .msu-popup {
+	display: none;
 	height: 72.0rem !important;
```

Comment:
Not a huge fan of `display: none;` being part of the CSS class, I think this should be a separate thing we flick on/off rather than a part of the class we override.

Reply:
All jquery hide() and show() does is toggle between None and whatever the last display was. I can remove it, but then I'll need to add a setState(None) somewhere during setup, it'll be the same thing.

Reply:
I would prefer that, this is a conceptual thing, the popup style isn't to be invisible, you can hide it, but that's not part of its class.

Reply:
Was deemed too annoying to do.